eventlet/support/greendns.py

```diff
@@ -77,15 +77,18 @@ def import_patched(module_name):
 DNS_QUERY_TIMEOUT = 10.0
 HOSTS_TTL = 10.0
 
-EAI_EAGAIN_ERROR = socket.gaierror(socket.EAI_AGAIN, 'Lookup timed out')
-EAI_NONAME_ERROR = socket.gaierror(socket.EAI_NONAME, 'Name or service not known')
+EAI_EAGAIN_ERRORARGS = (socket.EAI_AGAIN, 'Lookup timed out')
+EAI_NONAME_ERRORARGS = (socket.EAI_NONAME, 'Name or service not known')
+EAI_NONAME_ERROR = socket.gaierror(*EAI_NONAME_ERRORARGS)
```

Comment:
- (the only major comment here) hide exception args from public API (`_FOO_ARGS`)
- `FOO_ERROR_ARGS` or `FOO_ARGS`
- `EAI_EAGAIN_ERROR` was deleted, probably good idea to leave it like NONAME, NODATA errors.

```suggestion
_EAI_EAGAIN_ARGS = (socket.EAI_AGAIN, 'Lookup timed out')
EAI_EAGAIN_ERROR = socket.gaierror(*EAI_EAGAIN_ARGS)
_EAI_NONAME_ARGS = (socket.EAI_NONAME, 'Name or service not known')
EAI_NONAME_ERROR = socket.gaierror(*EAI_NONAME_ARGS)
```

Reply:
The desired change is in the lattest commit